python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -1444,3 +1454,7 @@ def info_string(self):
         autoscaler_summary = self.summary()
         assert autoscaler_summary
         return "\n" + format_info_string(lm_summary, autoscaler_summary)
+
+
+def _prioritize_and_filter_nodes(autoscaler: StandardAutoscaler) -> List[NodeType]:
```

Comment:
This node_selector neither prioritizes nor filters nodes. Maybe it could be called `default_node_selector` in a final version.
(I got the intention of "put a thing that prioritizes and filters here for better autoscaling behavior", it makes sense.)